lib/benchee/benchmark/collect/reductions.ex

```diff
@@ -0,0 +1,42 @@
+defmodule Benchee.Benchmark.Collect.Reductions do
+  @moduledoc false
+
+  @behaviour Benchee.Benchmark.Collect
+
+  def collect(fun) do
+    parent = self()
+
+    # The reduction offset here is important - this is the number of reductions
+    # that are needed for the other functions in our runner process that don't
+    # have anything to do with the function that's being benchmarked.
+    #
+    # It includes the one reduction for the function that we have wrapping the
+    # actual function being benchmarked, and then the rest are for the
+    # collection of the initial reduction count and the ending reduction count.
+    # These numbers vary based on the Erlang/OTP version, but not based on the
+    # Elixir version (based on my experiements of everything from OTP 19 - 22
+    # and Elixir 1.6 - 1.8).
+    reduction_offset =
+      case Benchee.System.erlang() do
```

Comment:
Not sure if it's great to hard code this, would it work to measure the overhead like we do for run time? https://github.com/bencheeorg/benchee/blob/master/lib/benchee/benchmark/runner.ex#L30-L35

The reason I'm not sure this is great is that it needs updates to work with new OTP versions (and a fall through) and I'm slightly scared that at some point it might also depend on the elixir version (new optimization) or in the worst case even OS (hopefully not!)

Reply:
This was a good idea! Turns out it was super easy to measure since it's a static value for a given run.